src/Servers/Kestrel/test/InMemory.FunctionalTests/TlsListenerMiddlewareTests.Units.cs

```diff
@@ -0,0 +1,534 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Move this file to src/Servers/Kestrel/Core/test? And remove `.Units`?

Reply:
this class uses `InMemoryTransportConnection` (convenient type to pass in data into middleware) which is defined in close-by namespace: `Microsoft.AspNetCore.Server.Kestrel.InMemory.FunctionalTests.TestTransport`

I just wanted to show that tests in this class are not using a server setup; but pass in data manually. I can simply remove `.Units` here and move it to the same class if you think that is more readable

Reply:
indeed; moved tests